classes/patterns/class-blankpage.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * BlankPage pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * This class is used for returning a blank page with a default content.
+ *
+ * @package P4GBKS\Patterns
+ */
+class BlankPage extends Block_Pattern {
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_name(): string {
+		return 'p4/blank-page';
+	}
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		return [
+			'title'      => __( 'Blank page', 'planet4-blocks-backend' ),
+			'blockTypes' => [ 'core/post-content' ],
+			'categories' => [ 'planet4' ],
```

Comment:
Let's use the `'pages'` category here, so it's grouped with the [rest of the page templates](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/877/files#diff-59777031749d62dcf9c1abe1512e44340868b40c3004277c5357267891d3bfecR33).